tests/console/libjpeg_turbo.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: libjpeg_turbo
+# Summary: Check libjpeg_turbo version
+# - Test should test on sle15sp6, see https://jira.suse.com/browse/PED-4889
+# - Check libjpeg-turbo version is higher than 2.1.1
+# - Do functional check with jpegtran command
+# Maintainer: QE Core <qe-core@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use version_utils;
+use utils qw(zypper_call upload_y2logs);
+use registration qw(add_suseconnect_product is_phub_ready);
+
+sub run {
+    select_serial_terminal;
+
+    # Package 'libjpeg-turbo' requires PackageHub is available
+    add_suseconnect_product('PackageHub') unless is_phub_ready();
```

Comment:
please add condition `is_sle` :)

Reply:
I changed to use ' return unless is_phub_ready();'